src/v2/guide/join.md

```diff
@@ -1,66 +1,66 @@
 
-## Resources
+## Sumber Daya
 
-### Code of Conduct
+### Kode Etik
 
-Our [Code of Conduct](/coc) is a guide to make it easier to enrich all of us and the technical communities in which we participate.
+[Kode Etik](/coc) kami adalah sebuah panduan untuk mempermudah kita semua dalam belajar dan komunitas yang membahas segala hal tentang teknis di tempat kita berpartisipasi.
 
-### Get Support
+### Dapatkan Bantuan
 
-- [Forum](https://forum.vuejs.org/): The best place to ask questions and get answers about Vue and its ecosystem.
-- [Chat](https://chat.vuejs.org/): A place for Vue devs to meet and chat in real time.
-- [Meetups](https://www.vuemeetups.org): Want to find local Vue.js enthusiasts like yourself? Interested in becoming a community leader? We have the help and support you need right here!
-- [GitHub](https://github.com/vuejs): If you have a bug to report or feature to request, that's what the GitHub issues are for. We also welcome pull requests!
+- [Forum](https://forum.vuejs.org/): Tempat yang tepat untuk menanyakan beberapa pertanyaan dan mendapatkan jawaban tentang Vue dan ekosistem nya.
+- [Chat](https://chat.vuejs.org/): Tempate Developer Vue untuk bertemu dan ngobrol secara real time.
+- [Meetups](https://www.vuemeetups.org): Ingin mencari penggemar Vuejs di lokal area seperti anda? Tertarik menjadi ketua komunitas? Kami disini bisa memberikan bantuan dan dukungan kepada anda !
+- [GitHub](https://github.com/vuejs): Jika anda menemukan `bug` untuk di laporkan dan meminta sebuah fitur baru di Vuejs, komunikasikan dengan kami melalui Github Issues. Kami juga menerima Pull Request dari Anda!
 
-### Explore the Ecosystem
+### Jelajahi Ekosistem
 
-- [The Awesome Vue Page](https://github.com/vuejs/awesome-vue): See what other awesome resources have been published by other awesome people.
-- [The "Show and Tell" Subforum](https://forum.vuejs.org/c/show-and-tell): Another great place to check out what others have built with and for the growing Vue ecosystem.
+- [Halaman Awesome Vue](https://github.com/vuejs/awesome-vue): Temukan sumberdaya / library keren lainnya yang telah di luncurkan oleh orang orang keren.
+- [The "Show and Tell" Subforum](https://forum.vuejs.org/c/show-and-tell): Tempat bagus lainnya untuk melihat apa yang telah dibuat oleh orang lain dan perkembangan Ekosistem Vuejs.
 
-## What You Can Do
+## Apa yang bisa anda lakukan
 
-### Contribute Code
+### Kontribusi kode
 
-As with any project, there are rules to contributing. To ensure that we can help you or accept your pull request as quickly as possible, please read [the contributing guide](https://github.com/vuejs/vue/blob/dev/.github/CONTRIBUTING.md).
+Seperti halnya project yang lain, ada beberapa peraturan dalam berkontribusi. Untuk memastikan kita bisa membantu atau menerima Pull Request dari anda secepat dan setanggap mungkin, dimohon untuk baca ini terlebih dahulu [the contributing guide](https://github.com/vuejs/vue/blob/dev/.github/CONTRIBUTING.md).
```

Comment:
Project -> projek

Reply:
Anda

Reply:
The contribution guide= Panduan berkontribusi

Reply:
Project cocok nya jadi proyek kali ya